### PR TITLE
new build: fix openmp issue

### PR DESCRIPTION
**Pull request by xoviat**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xoviat** [comment]
this build should resolve this issue: https://ci.appveyor.com/project/xianyi/openblas/builds/37531215/job/8bsnwgmaw3ahnjgh#L116887

note: this is not a regression from flang 5.0.0, because OpenMP was not working on 5.0.0.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {% set clang_variant = os.environ.get('CLANG_VARIANT', 'flang') %}
-{% set flang_commit = "ce9a7208dbb7b81ffeefd2eeeba055e8a37230cd" %}
+{% set flang_commit = "1192ff0775ddf94f7944dc40181406c088b4d575" %}
 {% set llvm_version = "11.0.1" %}
 {% set version = llvm_version %}
-{% set sha256 = "d2fb4555b1010ade283bccdd05d72593c5cec936d2c8eb932b62b81d563119ec" %}
-{% set build_number = "20200126" %}
+{% set sha256 = "e4fa46a0b29cabf4c4583231b79f923d6296efc4196099cbaebf5828f03a2607" %}
+{% set build_number = "20200131" %}
```
Comment (h-vetinari) [inline recipe/meta.yaml:6]: This should probably be
```suggestion
{% set build_number = "20210131" %}
```

**Comment by xoviat** [comment]
fix confirmed.

**Comment by xoviat** [comment]
@conda-forge-admin rerender

**Comment by xoviat** [comment]
cc @isuruf

**Pull request merged**